amundsen_application/static/js/components/common/AvatarLabel/index.tsx

```diff
@@ -22,7 +24,9 @@ const AvatarLabel: React.SFC<AvatarLabelProps> = ({ label, src }) => {
 
 AvatarLabel.defaultProps = {
   label: '',
-  src: ''
+  src: '',
+  link: '',
+  target: ''
```

Comment:
I think it makes sense to remove this part until we actually update the AvatarLabel and usage.

Reply:
Done, created OwnerAvatarLabelProps interface instead